k8s/namespaces/sscs/sscs-tribunals-api/aat.yaml

```diff
@@ -4,28 +4,14 @@ metadata:
   name: sscs-tribunals-api
   namespace: sscs
 spec:
-  releaseName: sscs-tribunals-api
-  rollback:
-    enable: true
-    retry: true
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/sscs-tribunals-api
   values:
     java:
-      replicas: 2
-      useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/sscs/tribunals-api:prod-c57e19c-20220506140005 #{"$imagepolicy": "flux-system:sscs-tribunals-api"}
```

Comment:
Is this needed in the AAT patch?

Reply:
`image:` I mean

Reply:
I wouldn't have thought so, however, the images used by some envs are different so I thought it better to include it in the overlay to specifically call out the version being used.

Reply:
Image moved to base chart